backend/app/routers/keycloak.py

```diff
@@ -188,27 +188,27 @@ async def refresh_token(
         )
 
 
-@router.get("/broker/{identity_provider}/token")
-def get_identity_provider_token(
-    identity_provider: str, access_token: str = Security(oauth2_scheme)
-) -> Json:
-    """Get identity provider JWT token from keyclok. Keycloak must be configured to store external tokens."""
-    if identity_provider in settings.keycloak_ipds:
-        idp_url = f"{settings.auth_base}/auth/realms/{settings.auth_realm}/broker/{identity_provider}/token"
-        idp_headers = {
-            "Content-Type": "application/x-www-form-urlencoded",
-            "Authorization": f"Bearer {access_token}",
-        }
-        idp_token = requests.request("GET", idp_url, headers=idp_headers)
-        # FIXME is there a better way to know if the token as expired and the above call did not go through?
-        idp_token.raise_for_status()
-        itp_token_body = json.loads(idp_token.content)
-        return itp_token_body
-    else:
-        raise HTTPException(
-            status_code=status.HTTP_400_BAD_REQUEST,
-            detail={
-                "error_msg": f"Identy provider [{identity_provider}] not recognized."
-            },
-            headers={"WWW-Authenticate": "Bearer"},
-        )
+# @router.get("/broker/{identity_provider}/token")
+# def get_idenity_provider_token(
```

Comment:
@longshuicy can you add a FIXME here saying we need to parse and return a consistent response? Also can you make an issue for it? I am afraid we will forget. Thank you.